contracts/KetherView.sol

```diff
@@ -1,29 +1,39 @@
 import "./IKetherHomepage.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 
-contract KetherView {
+library KetherView {
+  struct AdView {
+    address owner;
+    uint x;
+    uint y;
+    uint width;
+    uint height;
+    string link;
+    string image;
+    string title;
+    bool NSFW;
+    bool forceNSFW;
+    bool wrapped;
+  }
```

Comment:
is this beyond the one thats in EnumeratedNFT? As in will we ever want all your ads both wrapped and unwrapped because we might give you a wrap/unwrap interface?

Reply:
Yea probably wise to keep it separate. Main usecase I was thinking of is for a standalone wrapper UI that doesn't load the entire state and filter it.